subprojects/packagefiles/xtensor-python/meson.build

```diff
@@ -0,0 +1,47 @@
+project('xtensor-python', 'cpp',
+        version:'0.24.1',
+        license:'BSD-3-Clause',
+        default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+pybind_dep = dependency('pybind11',
+    version: ['>=2.4.3', '<3.0.0'])
+
+xtensor_dep = dependency('xtensor',
+    version: ['>=0.21.2', '<1.0.0'])
+
+python = import('python').find_installation('python3')
+
+npy_incstr = run_command(
+    [python, '-c', 'import numpy; print(numpy.get_include())'],
+    check: false).stdout().strip()
+
+if (npy_incstr == '')
+    warning('Unable to find numpy (Looking for it using ' + python.path() +
+        '), disabling python bindings.')
+endif
+
+extra_cflags = []
+
+xtensor_python_dep = declare_dependency(
+    include_directories: [include_directories('include', is_system: true),
+        include_directories(npy_incstr, is_system: true)],
+    dependencies:[xtensor_dep, pybind_dep])
+
+    extra_cflags += '-I' + npy_incstr
+
+# Workaround from mesonbuild/meson github issue 2550
+if meson.is_subproject()
+    install_dir = get_option('subproj_includedir')
+    extra_cflags += '-I' + get_option('prefix') / install_dir
+else
+    install_dir = get_option('includedir')
+endif
```

Comment:
Why is the install_dir different when it's a subproject?

Reply:
bc of [issue 2550](https://github.com/mesonbuild/meson/issues/2550)

Reply:
sorry, a better explanation would be that it is nice to have control over where a subproject gets installed. The resolution to issue `2550` was to allow the user to decide whether or not to install a subproject, but not to decide where to install it.

Reply:
@danielcjacobs whether or not to install a subproject in handled by the `--skip-subprojects` option on `meson install`

Reply:
Yes, but that doesn't allow you to control _where_ to install, just _whether_ to install

Reply:
Still don't understand why whether it's a subproject makes a difference to *where* to install.